File: app/src/main/java/net/xzos/UpgradeAll/ui/viewmodels/ViewHolder/LogRecyclerViewHolder.java
package net.xzos.UpgradeAll.ui.viewmodels.ViewHolder;

import android.view.View;
import android.widget.TextView;

import androidx.recyclerview.widget.RecyclerView;

public class LogRecyclerViewHolder extends RecyclerView.ViewHolder {
    public TextView logTextView;

    public LogRecyclerViewHolder(View view) {
        super(view);
        logTextView = view.findViewById(android.R.id.text1);
    }
}

File: app/src/main/kotlin/net/xzos/upgradeAll/ui/viewmodels/view/holder/LogRecyclerViewHolder.kt
package net.xzos.upgradeAll.ui.viewmodels.view.holder

import android.view.View
import android.widget.TextView

import androidx.recyclerview.widget.RecyclerView

class LogRecyclerViewHolder(view: View) : RecyclerView.ViewHolder(view) {
    var logTextView: TextView

    init {
        logTextView = view.findViewById(android.R.id.text1)
    }
}
